org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/NewXtextProjectWizard.java

```diff
@@ -97,6 +97,7 @@ protected IProjectInfo getProjectInfo() {
 		projectInfo.getSdkProject().setEnabled(advancedPage.isCreateSdkProject());
 		projectInfo.getP2Project().setEnabled(advancedPage.isCreateP2Project());
 		projectInfo.setLanguageServer(advancedPage.getLanguageServer());
+		projectInfo.setJunitVersion(advancedPage.getJUnitVersion());
```

Comment:
setJunitVersion vs getJUnitVersion (lowercase U vs Uppercase). Please keep it consistent.

Reply:
This resulted from WizardConfiguration:
```JUnitVersion junitVersion = JUnitVersion.JUNIT_4;```
I feel the field should be lower case. This results by Accessors annotation in a setter `setJunitVersion`.

However, in general JUnit is named with two capitals. So in AdvancedNewProjectPage it is
```
   def JUnitVersion getJUnitVersion () {
```

That results in this inconsistency. How should I solve that?

Reply:
maybe simply rename the the getter in AdvancedNewProjectPage?

Reply:
If you'd prefer...